docs/source/usingmame/commonissues.rst

```diff
@@ -242,4 +243,28 @@ What happened to the MAME support for external OPL2-carrying soundcards?
 
 MAME added support in version 0.23 for the use of a soundcard's onboard OPL2 (Yamaha YM3812 chip) instead of emulating the OPL2. This feature was never supported on the official Windows version of MAME and was dropped entirely as of MAME 0.60, because the OPL2 emulation in MAME had become advanced enough to be a better solution in almost all cases as of that time; now it's superior to using a sound card's YM3812 in all cases, especially as modern cards lack a YM3812.
 
-Unofficial builds of MAME may have supported it for varying amounts of time as well.
\ No newline at end of file
+Unofficial builds of MAME may have supported it for varying amounts of time as well.
+
+
+.. _Autofire:
+
+What happened to the MAME support for autofire?
+-----------------------------------------------
+
+Autofire was moved to a Lua script plugin in MAME 0.216. This new plugin has more functionality than the previous built-in autofire in MAME revisions previous to 0.216; for instance, you can specify an alternate button for the autofire.
```

Comment:
The plugin has been around for longer than that (since 0.210).  I'd change the wording to something like "A Lua plugin with providing enhanced autofire support was added in MAME 0.210; the built-in autofire functionality was removed in MAME 0.216.